image/templates/helm/shared/templates/_scanner-v4_init.tpl.htpl

```diff
@@ -163,7 +163,22 @@
 
 {{- end -}}
 
+{{- if eq $.Chart.Name "stackrox-secured-cluster-services" -}}
+  {{/* Special handling for the secured-cluster-services chart in case it gets deployed
+       to the same namespace as central-services. */}}
+  {{ $centralDeployment := dict }}
+  {{ include "srox.safeLookup" (list $ $centralDeployment "apps/v1" "Deployment" $.Release.Namespace "central") }}
+  {{ if $centralDeployment.result }}
+    {{ include "srox.note" (list $ "Detected central running in the same namespace. Not deploying scanner-v4-indexer from this chart and configuring sensor to use existing scanner-v4-indexer instance, if any.") }}
+    {{ $_ := set $._rox.sensor.localImageScanning "enabled" "true" }}
+    {{ $_ := set $components "indexer" false }}
+    {{/* Matcher on secured-cluster is unsupported, but just in case make sure it is disabled. */}}
+    {{ $_ := set $components "matcher" false }}
```

Comment:
Does this address the DB conflict? I may be missing where we disable creating the DB.

Reply:
we don't disable it. Instead we skip enabling it. :-)
In L185 below the DB is enabled, but only if scanner v4 components are actually to be deployed by the current chart.